app/models/manageiq/providers/google/cloud_manager/event_catcher/runner.rb

```diff
@@ -23,15 +23,9 @@ def monitor_events
     reset_event_monitor_handle
   end
 
-  def process_event(event)
-    if filtered?(event)
-      _log.info(
-        "#{log_prefix} Skipping filtered Google event #{parse_event_type(event)} for #{parse_resource_id(event)}"
-      )
-    else
-      _log.info "#{log_prefix} Caught event #{parse_event_type(event)} for #{parse_resource_id(event)}"
-      EmsEvent.add_queue('add_google', @cfg[:ems_id], event)
-    end
+  def queue_event(event)
+    _log.info "#{log_prefix} Caught event #{parse_event_type(event)} for #{parse_resource_id(event)}"
+    EmsEvent.add_queue('add_google', @cfg[:ems_id], event)
```

Comment:
either in this PR or in a follow up, you can remove the `add_google` methods with something like this https://github.com/durandom/manageiq-providers-amazon/blob/8e7fe72383976bc8f9d763ff757a53c53f9db1be/app/models/manageiq/providers/amazon/cloud_manager/event_catcher/runner.rb#L24-L24

this removes the coupling in `EmsEvent`